Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/azure/serverless/spark/models/CreateSparkBatchJobParameters.java

```diff
@@ -29,6 +29,7 @@
 
 /**
  * Parameters used to submit a new Data Lake Analytics spark batch job request.
+ * Some fields are commented since they already exists in the base class SparkSubmissionParameter
```

Comment:
`To leverage Livy(HDInsight) Spark Batch Job codes, some fields are commented since they can be inherited from the base class SparkSubmissionParameter`